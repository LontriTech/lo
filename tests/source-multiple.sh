#!/usr/bin/env bash

cd "${0%/*}" || exit 1
cat <<EOF | ../mo --source=source-multiple-1.vars --source=source-multiple-2.vars
A: {{A}}
B: {{B}}
C: {{C}}
EOF
