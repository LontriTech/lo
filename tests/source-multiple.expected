A: from1
B: from2
C: from2
